src/components/new-request/search.vue

```diff
@@ -1,43 +1,46 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal pa-10">
     <v-row no-gutters>
-      <v-col cols="12" class="pt-0 font-weight-bold h6"><span>I need a name to:</span></v-col>
+      <v-col cols="12" class="pt-0 font-weight-bold h6">
+        I need a name to: {{ request_action_cd }} / {{ location }} / {{ entity_type_cd }}
+      </v-col>
     </v-row>
 
     <v-row class="mt-4" no-gutters>
-      <!--request_action_cd-->
+      <!-- request_action_cd -->
       <v-col cols="12" md="4" lg="4">
-        <v-tooltip top
-                   id="search-type-options-select"
-                   content-class="top-tooltip"
-                   transition="fade-transition"
-                   :disabled="request_action_cd !== RequestCode.CNV || isMobile">
-          <template v-slot:activator="scope">
-            <div v-on="scope.on">
-              <v-select :error-messages="getErrors.includes('request_action_cd') ? 'Please select an action' : ''"
-                        :items="$requestActions"
-                        :menu-props="{ bottom: true, offsetY: true}"
-                        @change="clearErrors()"
-                        label="Select an Action"
-                        filled
-                        v-model="request_action_cd">
-                <template slot="item" slot-scope="data">
-                  <v-tooltip :disabled="!data.item.blurbs || isMobile" right transition="fade-transition">
-                    <template v-slot:activator="scope">
-                      <span v-on="scope.on" class="list-item">
-                        {{ data.item.text }}
-                      </span>
-                    </template>
-                    <span>{{ data.item.blurbs }}</span>
-                  </v-tooltip>
-                </template>
-              </v-select>
-            </div>
+        <v-select
+          filled
+          label="Select an Action"
+          :error-messages="getErrors.includes('request_action_cd') ? 'Please select an action' : ''"
+          :items="requestActions"
+          :menu-props="{ bottom: true, offsetY: true, maxHeight: 423 }"
+          @change="clearErrors()"
+        >
+          <template v-slot:item="{ item }">
+            <v-list-item-title
+              v-if="item.isHeader"
+              class="group-header d-flex justify-space-between align-center"
```

Comment:
What do you think of creating a class for this in the style block Sev? Would it be nicer?

Reply:
I think it would be worse, as I would have to code in all the CSS that the Vuetify helper classes provide here.